spec/mailers/devise_mailer_spec.rb

```diff
@@ -11,7 +11,7 @@
     end
 
     it "renders sender" do
-      expected_from = "#{SiteConfig.community_name} Community <#{SiteConfig.email_addresses[:default]}>"
+      expected_from = "#{SiteConfig.community_name} <#{SiteConfig.email_addresses[:default]}>"
       expect(email["from"].value).to eq(expected_from)
     end
```

Comment:
This is failing because `email["from"].value` returns `DEV (local) Community`, but I changed the value of `expected_from` to _not_ include the appended `Community` and I cannot figure out why `.value` still returns it _with_ the appended `Community` in the test. After looking into the mailer-related code and throwing `binding.pry`s in, I was unable to trace this `.value` back to anything that stood out as the culprit. Am I overthinking this? Should `Community` still remain appended in the spec? 😅 🤔